systemd/xmtpd-worker.service

```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description=xmtpd-worker
+After=network.target
+StartLimitIntervalSec=0
+
+[Service]
+Type=simple
+Restart=always
+RestartSec=1
+EnvironmentFile=/etc/xmtpd.env
+ExecStart=/usr/local/bin/xmtpd --indexer.enable --sync.enable --api.port=5060 --api.http-port=5065
```

Comment:
I dont think that you need the API ports for these two services. They don't expose anything other than possibly metrics and reflection. I will check whether you can run without them.

Reply:
gotcha! Filed it as an improvement https://github.com/xmtp/xmtpd/issues/1132